File: D155.c
#include <stdio.h>
#include <stdbool.h>
int main() {
    int ancho, alto;

    while (true) {
        scanf("%d %d", &ancho, &alto);
        if (ancho < 0 || alto < 0)
            break;

        int resultado = ancho * 2 + alto * 2;

        printf("%d\n", resultado);
    }
    return 0;
}

File: D155.java
import java.util.Scanner;

public class D155 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        while (true) {
            int ancho = sc.nextInt();
            int alto = sc.nextInt();
            if (ancho < 0 || alto < 0) break;
            System.out.println(ancho * 2 + alto * 2);
        }
    }
}
